tests/api/test_attributes.py

```diff
@@ -98,6 +98,35 @@ def test_attributes_query(user_api_client, product):
     assert len(attributes_data) == attributes.count()
 
 
+NOT_EXISTS_IDS_ATTRIBUTES_QUERY = """
+    query ($filter: AttributeFilterInput!) {
+        attributes(first: 5, filter: $filter) {
+            edges {
+                node {
+                    id
+                    name
+                }
+            }
+        }
+    }
+    """
+
+
+@pytest.mark.parametrize(
+    "attribute_filter", [{"ids": ["ygRqjpmXYqaTD9r=", "PBa4ZLBhnXHSz6v="]}],
+)
+def test_attributes_query_ids_not_exists(attribute_filter, user_api_client, category):
```

Comment:
Better to use `attribute_filter = {"ids": ["ygRqjpmXYqaTD9r=", "PBa4ZLBhnXHSz6v="]}`  insted of using `parametrize` with one parameter.

Reply:
Done, please take a look again :)